plugins/nomad-backend/README.md

```diff
@@ -1,6 +1,21 @@
 # @backstage/plugin-nomad-backend
 
-A backend for [Nomad](https://www.nomadproject.io/), this plugin exposes a service with routes that are used by the `@backstage/plugin-nomad` plugin to query Job and Group information from a Nomad API.
+A backend for [Nomad](https://www.nomadproject.io/), this plugin exposes a service with routes that are used by the `@backstage/plugin-nomad-backend` plugin to query Job and Group information from a Nomad API.
+
+## New Backend System
+
+The Nomad backend plugin has support for the [new backend system](https://backstage.io/docs/backend-system/), here's how you can set that up:
+
+In your `packages/backend/src/index.ts` make the following changes:
+
+```diff
+  import { createBackend } from '@backstage/backend-defaults';
+  import { nomadPlugin } from '@backstage/plugin-nomad-backend';
+  const backend = createBackend();
+  // ... other feature additions
++ backend.add(nomadPlugin());
```

Comment:
I think that these docs can be replaced with the example above:

```
backend.add(import('...'))
```

instead :)